chromeExtension/js/background.js

```diff
@@ -67,6 +62,19 @@ function makeUniquePageRequest(tab) {
   })
 }
 
+function postFetch(endRoute, method, body) {
```

Comment:
What if we make two functions - one for post requests and another for get requests. As a result, body wouldn't exist in the get request function.
'endRoute' should probably be called 'path'